Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8.5-alpine3.12
+MAINTAINER Mo'ath Zaghdad
+
+ENV PYTHONUNBUFFERD 1
+
+COPY ./requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
+
+RUN mkdir /app
```

Comment:
I didn't use it because I know it's not on the root directories of the image,
but you are right if I used a custom image it might cause an error if the file exists.

Reply:
The `-p`  flag is now added